lib/Server.js

```diff
@@ -122,7 +139,10 @@ function Server(compiler, options) {
 					this.io.sockets.emit("proxy-error", [msg]);
 				}.bind(this));
 			}.bind(this));
+
 		} else if(/^(https?:)?\/\//.test(contentBase)) {
+			console.log('Using contentBase as a proxy is deprecated and will be removed in the next major version. Please use the proxy option instead.\n\nTo update remove the contentBase option from webpack.config.js and add this:');
+			console.log('proxy: {\n\t"*": "'+contentBase+'"\n}');
```

Comment:
This is wrong here. `contentBase` is not used as proxy. It's used to redirect requests, which is not deprecated.